web/src/services/tracker.ts

```diff
@@ -146,6 +148,25 @@ export function trackChallenge(
   track('Challenge', action);
 }
 
+export function trackSingleSubmission(
+  action: 'toggle-button-click' | 'submit',
+  locale: string
+) {
+  track('SingleSubmission', action, locale);
+}
```

Comment:
Does `track` return something? Because then you could just do it like this:
```suggestion
export trackSingleSubmission = (
  action: 'toggle-button-click' | 'submit',
  locale: string
) => track('SingleSubmission', action, locale);
```
This way the caller can decide if they want to throw away the result or not. The same goes ofc for the other function.